webviewer/src/components/_shared/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { StyledBreadcrumbs, BreadcrumbsItem, Separator } from './Breadcrumbs.styles';
+import { Link } from '../Link';
+
+export const Breadcrumbs = ({ items, ...props }) => {
+  return (
+    <StyledBreadcrumbs {...props}>
+      {items.map(({ text, link }) => {
+        if (link)
+          return (
+            <BreadcrumbsItem key={text}>
```

Comment:
you have the same wrapper 

<BreadcrumbsItem key={text}>
</BreadcrumbItem>
maybe better add condition inside it, like 

`link ? <>link + separator</> : text`

Reply:
it will help you to have one return statement and only conditionally return needed content

Reply:
done